lib/account.rb

```diff
@@ -0,0 +1,55 @@
+require 'csv'
+
+module Bank
+  class Account
+    attr_reader :id, :balance, :openDate
+    def initialize(id, balance, openDate = '1999-03-27 11:30:09 -0800')
+      raise ArgumentError.new("balance must be >= 0") if balance < 0
+      @id = id
+      @balance = balance
+      @openDate = openDate
+    end
+
+    def self.all
+      all_accounts = [] #array of classes
```

Comment:
Good work on `Account.all` and `Account.find`, I feel like this implementation is about as clean as it gets.